### DISCUSSION TITLE
[Bug]: Slow asset creation using Datahub or 500 internal server error

### DISCUSSION THREAD

**Discussion by Wael-Hamdan** [pimcore/pimcore#10118] [Q&A / Support]
### Expected behavior

Create asset in a reasonable time.

### Actual behavior

Asset creation using Datahub takes 4s instead of a few milliseconds when using drag and drop of asset to upload asset in Pimcore directly.

Sometimes, the asset creation in Datahub fails and a 500 internal error message is received.

### Steps to reproduce

Convert image to bytes and encode it to base64 then send a mutation in Datahub as explained here ;
https://pimcore.com/docs/data-hub/current/GraphQL/Mutation/Asset_Mutations.html

**Comment by Wael-Hamdan** [comment]
Hi,
Maybe datahub is not adapted to upsert GBs of high quality images to pimcore. Is there any other alternative to datahub for this usecase ?

**Comment by cutec-chris** [comment]
Datahub is the only standard api included in pimcore. 
An system that behaves over an API bad is no good start for digitalisation.
